packages/appeal-reply-service-api/src/lib/config.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Config
+ *
+ * This is the single-source-of-truth for the application. All
+ * config should be driven by environment variables where different
+ * values are required
+ */
+
+const path = require('path');
```

Comment:
A lot of this file won't be needed

Reply:
Need some input on this. I've now deleted parts which I know won't be relevant. Please feel free to comment on anything else that could be removed.